extensions/ql-vscode/src/remote-queries/variant-analysis-manager.ts

```diff
@@ -40,29 +38,27 @@ export class VariantAnalysisManager extends DisposableObject implements VariantA
   public readonly onVariantAnalysisRemoved = this._onVariantAnalysisRemoved.event;
 
   private readonly variantAnalysisMonitor: VariantAnalysisMonitor;
-  private readonly variantAnalysisResultsManager: VariantAnalysisResultsManager;
   private readonly variantAnalyses = new Map<number, VariantAnalysis>();
   private readonly views = new Map<number, VariantAnalysisView>();
   private static readonly maxConcurrentDownloads = 3;
   private readonly queue = new PQueue({ concurrency: VariantAnalysisManager.maxConcurrentDownloads });
 
   constructor(
     private readonly ctx: ExtensionContext,
-    cliServer: CodeQLCliServer,
     private readonly storagePath: string,
-    logger: Logger,
+    private readonly variantAnalysisResultsManager: VariantAnalysisResultsManager
   ) {
     super();
     this.variantAnalysisMonitor = this.push(new VariantAnalysisMonitor(ctx));
     this.variantAnalysisMonitor.onVariantAnalysisChange(this.onVariantAnalysisUpdated.bind(this));
 
-    this.variantAnalysisResultsManager = this.push(new VariantAnalysisResultsManager(cliServer, logger));
+    this.variantAnalysisResultsManager = this.push(variantAnalysisResultsManager);
```

Comment:
I don't think we need to `push` this since the object is not managed by the variant analysis manager; we are already adding it as a disposable to `ctx.subscriptions` in the `extension.ts` file.

Reply:
Thanks @koesie10 ! Have changed.